src/components/Touchable.android.js

```diff
@@ -4,12 +4,23 @@ import { StyleSheet, TouchableNativeFeedback, View } from "react-native";
 import type { TouchableProps } from "./TouchableTypes";
 import { TouchableDefaultProps } from "./TouchableTypes";
 
-const Touchable = ({ children, style, ...props }: TouchableProps) => (
+const Touchable = ({
+  children,
+  style,
+  onPress,
+  disabled,
+  ...props
+}: TouchableProps) => (
   <TouchableNativeFeedback
+    accessibilityTraits={["button"]}
+    accessibilityComponentType="button"
     background={TouchableNativeFeedback.SelectableBackground()}
-    {...props}
+    onPress={onPress}
+    disabled={disabled}
   >
-    <View style={[styles.defaults, style]}>{children}</View>
+    <View style={[styles.defaults, style]} {...props}>
```

Comment:
The props currently contain all properties the `TouchableNativeFeedback` can handle. It's a bit weird to apply them to the view instead.

If there is a need to put extra props on the nested view, we might need to add a special prop for this. Or rethink the automatically added nested view here. I put it there because there are restrictions for the children of the `TouchableNativeFeedback`:
> At the moment it only supports having a single View instance as a child node, as it's implemented by replacing that View with another instance of RCTView node with some additional properties set.

In any case, I think these props should be applied to the `TouchableNativeFeedback`.